install/page_dbconnection.php

```diff
@@ -46,8 +46,8 @@
 
 $error = '';
 if ($_SERVER['REQUEST_METHOD'] == 'POST' && !empty( $vars['DB_HOST'] ) && !empty( $vars['DB_USER'] )) {
-	$func_connect = empty( $vars['DB_PCONNECT'] ) ? "mysql_connect" : "mysql_pconnect";
-	if (! ( $link = @$func_connect( $vars['DB_HOST'], $vars['DB_USER'], $vars['DB_PASS'], true ) )) {
+    $hostname = empty( $vars['DB_PCONNECT'] ) ? $vars['DB_HOST'] : "p:".$vars['DB_HOST'];
+    if (! ( $link = @mysqli_connect( $hostname, $vars['DB_USER'], $vars['DB_PASS']) )) {
 		$error = ERR_NO_DBCONNECTION;
```

Comment:
You should be able to put in print mysqli_error() at this point, or some function like that, which will tell us the error from MySQL, such as invalid host or something. What you're seeing on screen now I think is the installer itself saying there's no connection, but we need to see why.